sklearn/preprocessing/imputation.py

```diff
@@ -102,10 +104,15 @@ class Imputer(BaseEstimator, TransformerMixin):
         - If `axis=0` and X is encoded as a CSR matrix;
         - If `axis=1` and X is encoded as a CSC matrix.
 
+    n_neighbors : int, optional (default=1)
+        It only has effect if the `strategy=knn`. It controls the number of
```

Comment:
Sure. Should I also change other comments using backticks? Like:
 - If `axis=0` and X is encoded as a CSR matrix;
 - If `axis=1` and X is encoded as a CSC matrix.